### PR TITLE
update arrow key behavior

### PR DESCRIPTION
**Pull request by TheMarit**
To me it makes more sense if the active cell doesn't move on arrow keypress when you are actively editing the cell. This way you can navigate through the text you are typing.

### CODE REVIEW

File: src/components/VueTable.vue [hunk 1]
```diff
@@ -1311,8 +1311,9 @@ export default {
           }
           this.$set(this.tbodyData[rowIndex][header], 'active', false);
           this.removeClass(['rectangleSelection']);
+          const currentlyEditingCell = this.tbodyData[rowIndex][header].show;
```
Comment (joffreyBerrier) [inline src/components/VueTable.vue:1314]: It work's, but i think it's better to put this condition on the top of the function. I need to look that. I'll come back to you when i look that in deepness. I'll try to look at that on the afternoon !

Comment (joffreyBerrier) [reply src/components/VueTable.vue:1314]: Put your condition like this =>

```
const currentlyEditingCell = this.tbodyData[rowIndex][header].show;

if (!currentlyEditingCell) {
  if (!this.setFirstCell) {
    ...
  }
  ...
  if (this.keys[16]) {
    ...
  } else if (!this.lastSelectOpen && event.keyCode !== 8) {
    ...
  }
}
```

Comment (TheMarit) [reply src/components/VueTable.vue]: Ok i've updated this

**Comment by TheMarit** [comment]
@joffreyBerrier I've updated the changes you requested. Please have a look and let me know if there is anything else.

**Pull request closed without merging**